matrix.py

```diff
@@ -0,0 +1,55 @@
+from typing import Self
+
+class Matrix:
+    """
+    Matrix class
+
+    Represents a mathematical matrix
+
+    """
+
+    def __init__(self: Self, columns: list[list[int | float]]) -> None:
+        """init method"""
```

Comment:
Dont forget the naming conventions.
```suggestion
"""Init method."""
```
One line comment must end with a point  **.**